GithubPractice/ViewController.swift

```diff
@@ -29,4 +35,61 @@ class ViewController: UIViewController {
     @IBAction func didClickClear(_ sender: Any) {
         
     }
+    
+    
+    override func didReceiveMemoryWarning() {
+          super.didReceiveMemoryWarning()
+          // Dispose of any resources that can be recreated.
+      }
+
+  //This is for the keyboard to GO AWAYY !! when user clicks anywhere on the view
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        self.view.endEditing(true)
+    }
+
+  //This is for the keyboard to GO AWAYY !! when user clicks "Return" key  on the keyboard
+
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+          textField.resignFirstResponder()
```

Comment:
A good practice is to separate delegates, such as `UITextFieldDelegate` in an extension. All the methods related to that delegates should to be implemented there.

Example:
`extension ViewController: UITextFieldDelegate { ... }`